db/seeds.d/07-provisioning_templates.rb

```diff
@@ -1,14 +1,14 @@
 # Find known operating systems for associations
-os_junos = Operatingsystem.find_all_by_type "Junos" || Operatingsystem.where("name LIKE ?", "junos")
-os_solaris = Operatingsystem.find_all_by_type "Solaris"
-os_suse = Operatingsystem.find_all_by_type "Suse" || Operatingsystem.where("name LIKE ?", "suse")
-os_windows = Operatingsystem.find_all_by_type "Windows"
+os_junos = [Operatingsystem.where(:type => "Junos").first || Operatingsystem.where("name LIKE ?", "junos").first].compact
+os_solaris = [Operatingsystem.where(:type => "Solaris").first].compact
+os_suse = [Operatingsystem.where(:type => "Suse").first || Operatingsystem.where("name LIKE ?", "suse").first].compact
+os_windows = [Operatingsystem.where(:type => "Windows").first].compact
 
 # Template kinds
 kinds = {}
 [:PXELinux, :PXEGrub, :iPXE, :provision, :finish, :script, :user_data, :ZTP, :POAP].each do |type|
-  kinds[type] = TemplateKind.find_by_name(type)
```

Comment:
Yeah, section 4.4 (http://guides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-3-2-to-rails-4-0-active-record) says "All dynamic methods except for find_by_... and find_by_...! are deprecated".